src/main/java/org/reactome/orthoinference/SkipInstanceChecker.java

```diff
@@ -23,14 +17,17 @@
 	private static final Logger logger = LogManager.getLogger();
 	private static MySQLAdaptor dba;
 	private static Set<String> skipList = new HashSet<>();
+	private static final long HIV_INFECTION_DBID = 162906L;
+	private static final long INFLUENCE_INFECTION_DBID = 168255L;
```

Comment:
Typo - INFLUENZA?

Reply:
Wow, thats pretty funny. It's a condition that makes you start a youtube channel

Reply:
Fixed